app/decorators/ncr/work_order_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Ncr
+  class WorkOrderDecorator < Draper::Decorator
+    delegate_all
+
+    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
+    NO_APPROVER_FOUND        = 'No Approver Found'
+
+    def current_approver_email_address
+      approver_email_address(current_approver)
+    end
+
+    def final_approver_email_address
+      approver_email_address(final_approver)
+    end
+
+    private
+
+    def approver_email_address(approver)
+      if approver
+        approver.email_address
+      elsif !approver and self.emergency
```

Comment:
I think this can just be `emergency` since L19 means that anything in this `elsif` would not have an approver